contents/tutorials/hogql-sum-aggregation.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "The power of HogQL’s sum() aggregation"
+date: 2023-06-06
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-8.png
+tags: ['hogql', 'trends', 'insights']
+---
+
+HogQL enables custom series aggregations. This enables you to customize the data filtered, visualized, and broken down in your insights. One of the most powerful aggregations is `sum()`, which we will showcase in this tutorial.
+
+## The basics of sum()
+
+Sum is simple, it calculates the sum of the numbers it is provided. In its most basic form, we can use this to sum event properties (or nested one). For example, to sum your revenue, chose your payment event, then sum the amount paid property. The HogQL expression for this is `sum(properties.amount_paid)`
+
+![Sum](../images/tutorials/hogql-sum-aggregation/sum.png)
+
+> **Note:** sum only works for numbers, you can use `toInt()` to convert any strings to numbers.
+
+## Basic conditional sum
+
+Sum gets more interesting when we use it with conditions. A way to think about this is that "Total count" (or`count()` in HogQL) is just `sumIf(1, true)`. 
+
+From this, we can create sums that meet certain conditions. For example, if we wanted a count of events during the current month, we could do:
+
+```
+sumIf(
+	1, 
+	toStartOfMonth(timestamp) = toStartOfMonth(now())
+)
+```
+
+This lets us sum occurrences of properties, properties that match criteria, and more.
+
+## Scoring criteria
+
+A more creative way to use `sumIf()`  is to give different weights to different values. 
+
+For example, if you wanted to rank sign up events based on specific criteria, you could use a chain of  `sumIf` and different values for different properties. For example, we can sum the `user signed up` event, and give:
+
+- 10 points for `is_organization_first_user`
+- 5 for `realm = ‘cloud’`
+- -1 if `team_member_count_all > 1`
+
+As a HogQL expression, this looks like:
+
+```
+sumIf(
+	10, 
+	properties.is_organization_first_user
+) + sumIf(
+	5,
+	properties.realm = 'cloud'
+) + sumIf(
+	-1,
+	properties.team_member_count_all > 1
+)
+```
+
+When we breakdown by `person_id`, this gives us a list of users with the highest scores, and we can customize our score this as our definition of "high-potential user" changes.
+
+![Score](../images/tutorials/hogql-sum-aggregation/score.png)
+
+> **Note:** person properties can’t be used with HogQL expression series, only event properties.
+
+## Sum to percentages
+
+Sum provides more options for percentages, because we can access both the total sum and a subset in the same series. This enables us to create percentage series.
+
+For example, if you wanted to compare the percentages of Chrome vs Safari, you can create a series where your divide the Safari pageviews by the total Safari and Chrome pageviews. 
```

Comment:
> to compare the percentages of Chrome vs Safari

Percentages of what? It's not clear what we're trying to achieve here, so I wasn't able to follow along with the example

Reply:
reworded